demisto_sdk/commands/common/mdx-parse-server.js

```diff
@@ -1,6 +1,54 @@
 const mdx = require('@mdx-js/mdx');
 const http = require('http')
+const markdownlint = require('markdownlint')
+const markdownlintRuleHelpers = require('markdownlint-rule-helpers')
 
+// explanation of the config can be found at
+// https://github.com/DavidAnson/markdownlint/blob/main/schema/markdownlint-config-schema.json
+const config = require("./markdownlintconfig.json")
+
+function getParamsFromReq(req) {
+    let split = req.url.split('?')
+    if (split.length < 2)
+        return {}
+    let params = {}
+    for (let param of split[1].split('&')) {
+        let vals = param.split('=')
+        params[vals[0]] = !vals[1] || 'false' == vals[1].toLowerCase() ? false : vals[1]
+    }
+    return params
+}
+function markdownLint(req, res, body) {
+
+    res.statusCode = 200
+    params = getParamsFromReq(req)
```

Comment:
Instead of doing a lot of dirty work on your own, it is better to use the express builtin features like `request.query`. See https://www.educative.io/answers/what-is-reqquery-in-expressjs
```suggestion
    params = req.query
```